### DISCUSSION TITLE
meetup Agile Munich (3200+ members) hosting a GSF session: how?

### DISCUSSION THREAD

**Discussion by peter-jetter** [Green-Software-Foundation/discussions#83] [Green Software Meetups] [answered]
Hi, i am a co-host of meetup Agile Munich since 2014. We have some 3200 members and roughly monthly sessions with about 100 participants.
 I would love to host a session about GSF. I do no want our meetup to move under the control of GSF. Sponsoring is nice, but not worth giving up our autonomy.
 Is there a way to run a session about GSF without having to become a GSF-owned meetup? eg GSF being a co-host/panelist/speaker for that one session only.

**Comment by NAMRATA-WOKE** [comment]
@adamj89

**Comment by adamj89** [comment, marked as answer]
Hi Peter! 

It's great that you'd like to host a session about GSF. The main staff of the foundation is small, fewer than 10 people and none in Germany. But I'm pleased to say that many of our members and GSF community do live in the Munich area.

On the community side please do speak with @mkorbi who is organising the Meetup alongside @Alexander-Kroll.  Also paging @ines-akrap and @cthornewill who are both heavily involved in the Green Software movement. 

Our Munich meetup is keen to get up-and-running, perhaps it would be a good opportunity to do a joint event?

**Reply by adamj89** [reply]
Also adding @by-d-sign who is leading another Green Software group in Munich.
